From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Tu Dinh <ngoc-tu.dinh@vates.tech>
Date: Sat, 7 Mar 2026 16:57:21 +0100
Subject: [PATCH] Add build location hack for burn

Signed-off-by: Tu Dinh <ngoc-tu.dinh@vates.tech>

diff --git a/src/burn/Directory.Build.props b/src/burn/Directory.Build.props
index 6050781df732..40e24661e211 100644
--- a/src/burn/Directory.Build.props
+++ b/src/burn/Directory.Build.props
@@ -7,8 +7,8 @@
   </PropertyGroup>
 
   <Import Project="..\Directory.Build.props" />
-  
+
   <PropertyGroup>
-    <BurnGeneratedHeaderDirectory>$(BaseIntermediateOutputPath)..\</BurnGeneratedHeaderDirectory>
+    <BurnGeneratedHeaderDirectory>$(RootBuildFolder)$(SegmentName)\obj\$(Configuration)\$(PlatformFolder)\</BurnGeneratedHeaderDirectory>
   </PropertyGroup>
 </Project>
