From 58e055383829eef4b2ba551bd01198834fb07faa Mon Sep 17 00:00:00 2001
From: Tu Dinh <ngoc-tu.dinh@vates.tech>
Date: Sat, 7 Mar 2026 16:57:21 +0100
Subject: [PATCH] Add build location hack for burn

Signed-off-by: Tu Dinh <ngoc-tu.dinh@vates.tech>
---
 src/burn/Directory.Build.props | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/burn/Directory.Build.props b/src/burn/Directory.Build.props
index 6050781d..40e24661 100644
--- a/src/burn/Directory.Build.props
+++ b/src/burn/Directory.Build.props
@@ -7,8 +7,8 @@
   </PropertyGroup>
 
   <Import Project="..\Directory.Build.props" />
-  
+
   <PropertyGroup>
-    <BurnGeneratedHeaderDirectory>$(BaseIntermediateOutputPath)..\</BurnGeneratedHeaderDirectory>
+    <BurnGeneratedHeaderDirectory>$(RootBuildFolder)$(SegmentName)\obj\$(Configuration)\$(PlatformFolder)\</BurnGeneratedHeaderDirectory>
   </PropertyGroup>
 </Project>
